packages/core/src/components/DatePickerInput/index.tsx

```diff
@@ -103,7 +105,7 @@ export default class DatePickerInput extends React.Component<
       locale: locale ?? this.props.locale,
       noTime: true,
       noTimezone: true,
-    });
```

Comment:
This seems like a trap.

Reply:
yeah it made ts happy 😬

Reply:
Is the original type nullable?

Reply:
ah, i can remove this! an artifact of a different approach